build-release-tools/application/jump_version.py

```diff
@@ -83,24 +108,23 @@ def update_changelog(self, message=None):
         if not debian_exist:
             # Handle repository which contains debianstatic/repository_name folder, 
```

Comment:
this parts of annotations is the same with link_debianstatic, it could be removed, or refined